php/Blockonomics.php

```diff
@@ -297,11 +297,19 @@ public function testSetup()
     }
 
     // Create the order url to redirect the user to during checkout
```

Comment:
When I read the function comment, it was previously clear what the function was doing. Now with the *select* parameter it is completely confusing as the function comment and working doesn't match. I had to parse code for 10min to understand the logic. We need  functions to be modular - this means the input, output and working of the function should be understood just by reading the function itself. 

Here is a better structure:
```

get_order_checkout_url()
 // Returns url to redirect the user to during checkout
```

```
get_crypto_checkout_url($crypto, $order_id)
//Returns order url for checkout using a given cryptocurrency 
```

Reply:
Updated in 8e57418

Reply:
Refactored code in 483b329

Reply:
Further refactored functions to be more usable in 412bc0b